### DISCUSSION TITLE
Adaptive Trigger Support for PS 5 controllers

### DISCUSSION THREAD

**Discussion by shaunkoh** [LizardByte/community#366] [Sunshine] [answered]
Support for Adaptive Triggers would be awesome. It’s one of the hallmark features of the PS 5 controller. I did a quick search of the project’s issues and discussions, but oddly couldn’t find any earlier mention of this. 

Consider this a +1

**Comment by ReenigneArcher** [comment, marked as answer]
Added in https://github.com/LizardByte/Sunshine/pull/5368
